docs/pages/omnia3/Modeler/omnia3_modeler_components.md

```diff
@@ -0,0 +1,189 @@
+
+A common scenario for this component is to execute a query that feeds a list.
+
+- Properties:
+
+| Property     | Direction | Data Type       |                                                                                       | Description |
+| ------------ | --------- | --------------- | ------------------------------------------------------------------------------------- | ----------- |
+| query        | Inbound   | Reference/Query | The modeled query to be executed                                                      |
+| records      | Outbound  | Object          | The records obtained as the result of the query execution                             |
+| columns      | Outbound  | Object          | The structure of the columns returned on each record                                  |
+| dataSource   | Inbound   | Text            | The dataSource instance where the query is going to be executed                       |
+| parameters   | Inbound   | Object          | Parameters to be applied when executing the query                                     |
+| filters      | Inbound   | Object          | The filters to be applied to the query                                                |
+| sorting      | Inbound   | Object          | The sorting criteria to be applied                                                    |
+| currentPage  | Inbound   | Integer         | The number of the page being retrieved                                                |
+| pageSize     | Inbound   | Integer         | The number of records to be returned                                                  |
+| totalPages   | Outbound  | Integer         | The total number of pages that can be obtained for the current query and page size    |
+| totalRecords | Outbound  | Integer         | The total number of records that can be returned as the result of the query execution |
+| isLoading    | Outbound  | Boolean         | Boolean to indicate if the query is being executed and data is still loading          |
+
+- Events:
+
+| Event       | Description                                                          |
+| ----------- | -------------------------------------------------------------------- |
+| BeforeLoad  | Code executed before making the request to the API                   |
+| OnLoad      | Code executed when data is retrieved from API, before being rendered |
+| OnLoadError | Code executed when the request to retrieve data returns an error     |
+
+- Methods:
+
+| Method    | Description                                          |
+| --------- | ---------------------------------------------------- |
+| refresh   | Method that executes the query to retrieve new data  |
+| exportCSV | Method that requests to the API a data export to CSV |
+
+### entityForm
+
+The entityForm Component renders a Form composed by other Components inside a page.
+
+A common scenario for this component is to show a Form to create/edit records.
```

Comment:
The entityForm is a functional Component use to manipulate modeling entities data (create/update/read/delete).